spikeinterface/postprocessing/correlograms.py

```diff
@@ -236,35 +190,47 @@ def compute_correlograms(waveform_or_sorting_extractor,
                                      bin_ms=bin_ms, symmetrize=symmetrize,
                                      method=method)
 
-def _compute_correlograms(sorting, window_ms: float = 100.0,
-                          bin_ms: float = 5.0, symmetrize: bool = False,
-                          method: str = "auto"):
+
+def _compute_correlograms(sorting, window_ms, bin_ms, symmetrize=None, method="auto"):
```

Comment:
Is having a parameter `window_ms` instead of `max_time` make sense?

I feel it's more intuitive to give the positive border rather than the size of the window (in the case of `symmetrize=True`)

Reply:
too late to much code depend on this.

Reply:
Ok.